### ISSUE TITLE
Lacks test for attributes in twinglydata element

### ISSUE THREAD

**Issue by walro** [twingly/twingly-search-api-python#26]
We have no test that checks that we correctly parse and extract the numbers from the `twinglydata` element:

``` xml
<twinglydata numberOfMatchesReturned="3" secondsElapsed="0.148" numberOfMatchesTotal="3">
```

**Comment by xSAVIKx** [comment]
I think current issue could be closed with #31 also.

Please check [following lines](https://github.com/twingly/twingly-search-api-python/blob/master/tests/test_parser.py#L57-L60) in tests.

**Comment by roback** [comment]
I'll close this as it was fixed by @xSAVIKx in #32 🎉
